api/src/controllers/schema.ts

```diff
@@ -101,12 +98,23 @@ const schemaMultipartHandler: RequestHandler = (req, res, next) => {
 };
 
 router.post(
-	'/diff',
+	'/apply',
```

Comment:
This suggestion doesn't really affect any code/logic, but in hindsight, let's also move this `/apply` route under the `/diff` route, just as a "mental model" thing to match the intended order of `snapshot, diff, apply` 😄

Reply:
oh I see, yup that makes sense :D